### PR TITLE
Issue#3589 check for raw in signal handlers 

### PR DESCRIPTION
**Pull request by salty-ivy**
_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [x] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

for issue https://github.com/wagtail/wagtail/issues/3589

okay okay so initially there's only this problem whenever  we try to load data with fixtures and if  WAGTAILIMAGES_FEATURE_DETECTION_ENABLED=True , pre_save_image_feature_detection signal handler used to throw an error
if the WAGTAILIMAGES_FEATURE_DETECTION_ENABLED=True and image file described in fixture doesn't exists 
image
after the check pre_save signal now check for raw==true for fixtures .
image

basically i wrote the test which creates the condition, where  WAGTAILIMAGES_FEATURE_DETECTION_ENABLED is True and source image described via fixture doesn't exist,  initially it fails when pre_save_image_feature_detection triggers because there wasn't any check for raw which indicated loading from fixtures, and in that case when images is not found it fails to load the fixture and throws an error, added check for "raw is false" , makes the signal runs only if raw is false when not from fixture, and if its true (in this case pre_save_image_feature_detection shouldn't do anything as that no error occurs)
signal doesn't fail, and still loads the fixture as demanded  by the issue

### CODE REVIEW

**Comment by 4the4ryushin** [comment]
okay okay so initially there's only this problem whenever  we try to load data with fixtures and if  WAGTAILIMAGES_FEATURE_DETECTION_ENABLED=True , pre_save_image_feature_detection signal handler used to throw an error
if the WAGTAILIMAGES_FEATURE_DETECTION_ENABLED=True and image file described in fixture doesn't exists 
image
after the check pre_save signal now check for raw==true for fixtures .
image

**Comment by Scotchester** [comment]
@4the4ryushin Thank you for your proposed contribution! Would you please update the PR title and description to describe what the changes do, beyond fixing issue #3589. I don't know what issue #3589 was, and there isn't even a link to it in the description for me to easily follow.

**Comment by 4the4ryushin** [comment]
> @4the4ryushin Thank you for your proposed contribution! Would you please update the PR title and description to describe what the changes do, beyond fixing issue #3589. I don't know what issue #3589 was, and there isn't even a link to it in the description for me to easily follow.

@Scotchester  i am extremely sorry i made the comment describing the issue instead of writing it in the description ,
i updated it

**Comment by 4the4ryushin** [comment]
i would like to raise one more point here as issue says we need to skip and do nothing when loading from fixtures , but as i see it
set_focal_point must be run successfully on every image regardless of where loads from so at raw == true , from fixtures it should also set the focal_point but as tried with original existing image, via fixture i ran into this issue .
image

**Review by lb- — APPROVED** [review]
@salty-ivy thank you so much for taking the time to investigate and fix this. Thanks also for writing a unit test that clearly reproduces the problem so that we can avoid this bug re-appearing in the future.

Also, it will be great to get a ~5 year old bug fixed!

**Comment by salty-ivy** [comment]
> @salty-ivy thank you so much for taking the time to investigate and fix this. Thanks also for writing a unit test that clearly reproduces the problem so that we can avoid this bug re-appearing in the future.
> 
> Also, it will be great to get a ~5 year old bug fixed!

The pleasure is all mine, love to contribute to wagtail
